includes/class-wc-round-tax.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * WooCommerce tax calculation class.
+ *
+ * @class   WC_Round_Tax
+ * @package WooCommerce/Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Round_Tax class.
+ */
+class WC_Round_Tax {
+	/**
+	 * Precision.
+	 *
+	 * @var int
+	 */
+	protected $precision = 0;
+
+	/**
+	 * Tax inclusive.
+	 *
+	 * @var bool
+	 */
+	protected $inclusive = false;
+
+	/**
+	 * Tax rate.
+	 *
+	 * @var float
+	 */
+	protected $tax_rate = 0.0;
```

Comment:
will probably need to support multiple tax rates with compounding

Reply:
> Are you also planning to add a new class for calculating totals and combining different tax and subtotal values as well?

That seems like the best approach.